code/__HELPERS/_string_lists.dm

```diff
@@ -1,27 +1,40 @@
 #define pick_list(FILE, KEY) (pick(strings(FILE, KEY)))
+#define pick_list_replacements(FILE, KEY) (strings_replacement(FILE, KEY))
 
 var/global/list/string_cache
+//This is a hack to track which file is being processed for string subkey
+//replacements, yes it's ugly, I don't know an easier way though
+var/global/string_filename_current_key
+
+/proc/strings_replacement(filename, key)
+	load_strings_file(filename)
+
+	if((filename in string_cache) && (key in string_cache[filename]))
+		var/response = pick(string_cache[filename][key])
+		var/regex/r = regex('@pick\\((\\D+)\\)', 'g')

```

Comment:
this regex is equivalent to 
    @pick(\D+)
with the \D+ being captured (any non whitespace or digit symbol)